demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -284,12 +327,17 @@ def _search_packs_by_items_names_or_ids(items_names: Union[str, list],
             for item_from_id_set in items_list:
                 item_id = list(item_from_id_set.keys())[0]
                 item_details = list(item_from_id_set.values())[0]
-                if (item_id in item_possible_ids or item_name == item_details.get('name')) \
-                        and item_details.get('pack') \
-                        and LooseVersion(item_details.get('toversion', DEFAULT_CONTENT_ITEM_TO_VERSION)) >= \
-                        MINIMUM_DEPENDENCY_VERSION \
-                        and (item_details['pack'] not in constants.IGNORED_DEPENDENCY_CALCULATION or
-                             not exclude_ignored_dependencies):
+                id_set_item_aliases = set(item_details.get('aliases', []))
+
+                is_item_id_match = item_id in item_possible_ids or item_name == item_details.get('name')
+                if item_type == 'incidentfield':
```

Comment:
Should be "incident_field" instead of "incidentfield"?

Reply:
This is the value passed by the calling function.
Not sure what will break if we change it, and I don't want to find out.